account_credit_control/views/credit_control_line.xml

```diff
@@ -0,0 +1,191 @@
+                    <filter name="group_by_policy_level" domain='[]'
+                            context="{'group_by': 'policy_level_id'}"
+                            icon="terp-document-new"
+                            string="Credit policy level"/>
+                    <filter name="group_by_channel" domain='[]' context="{'group_by': 'channel'}"
+                            icon="terp-document-new" string="Channel"/>
+                    <filter name="group_by_overridden" domain='[]'
+                            context="{'group_by': 'manually_overridden'}"
+                            icon="terp-document-new" string="Manual change"/>
+                    <filter name="group_by_followup" domain='[]'
+                            context="{'group_by': 'manual_followup'}"
+                            icon="terp-document-new" string="Manual followup"/>
+                    <filter name="group_by_partner_user" string="Salesperson" context="{'group_by':'partner_user_id'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="credit_control_line_tree" model="ir.ui.view">
+        <field name="name">credit.control.line.tree</field>
+        <field name="model">credit.control.line</field>
+        <field name="arch" type="xml">
+            <tree string="Control Credit Lines" editable="bottom"
+                  decoration-danger="state in ('error', 'email_error')"
+                  decoration-muted="state == 'ignored'"
+                  decoration-success="state == 'sent'">
+                <button name="button_credit_control_line_form"
+                        aria-label="Control Credit Line Form" title="Control Credit Line Form"
+                        type="object" icon="fa-external-link" class="oe_stat_button"/>
+                <field name="date"/>
+                <field name="date_due"/>
+                <field name="level"/>
+                <field name="manually_overridden"/>
+                <field name="state"/>
+                <field name="channel"/>
+                <field name="invoice_id"/>
+                <field name="partner_id"/>
+                <field name="manual_followup"/>
+                <field name="amount_due"/>
+                <field name="balance_due"/>
+                <field name="currency_id"/>
+                <field name="move_line_id"/>
+                <field name="account_id"/>
+                <field name="policy_level_id"/>
+                <field name="policy_id"/>
+                <field name="mail_message_id"/>
+                <field name="activity_date_deadline"/>
+                <button name="button_schedule_activity" type="object"
+                        aria-label="Schedule activity" title="Schedule activity" icon="fa-clock-o"/>
+                <field name="activity_summary"/>
+            </tree>
+        </field>
+    </record>
+
+    <menuitem name="Credit Control"
+              parent="account.menu_finance"
```

Comment:
In EE this will create a new menu named "invoicing"

Reply:
We can't do anything about this. Odoo decided to include EE menu only and relocate all the entries. The only solution is a glue module for fixing this.